index.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const Joi = require('joi');
+const emailer = require('./email');
+
+// Joi Schema Validation
+const SCHEMA_ADDRESS = Joi.string().email();
+const SCHEMA_ADDRESSES = Joi.array()
+    .items(SCHEMA_ADDRESS)
+    .min(0);
+const SCHEMA_STATUS = Joi.string();
```

Comment:
You should probably validate the values, too.
```
Joi.string().values(Object.keys(DESCRIPTION_MAP));
```

Remember that `DESCRIPTION_MAP` should declared be in source order before it is used.